akka-interop/src/main/scala/zio/actors/akka/AkkaTypedActorRefLocal.scala

```diff
@@ -0,0 +1,50 @@
+package zio.actors.akka
+
+import akka.actor.typed
+import akka.actor.typed.Scheduler
+import akka.actor.typed.scaladsl.AskPattern._
+import akka.util.Timeout
+import zio.{ Task, UIO, ZIO }
+
+/**
+ *
+ * Reference to proxy actor that contains an akka typed actor ref
+ *
+ * @tparam F wrapper type constructing DSL
+ */
+private[actors] final class AkkaTypedActorRefLocal[-F[+_]](actorName: String, akkaActor: typed.ActorRef[F[_]])
```

Comment:
As it will be public API used by end-users let's drop `private[actors]` and add `private val` to passed arguments: `private val actorName: String` etc.